tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+from .utilits import name_generator
+
+
+@pytest.fixture
+def customer_data(last_name="Иванов", post_code="0001252667"):
+    first_name = name_generator(post_code)
+    return {
+        "first_name": first_name,
+        "last_name": last_name,
+        "post_code": post_code,
+    }
```

Comment:
Можно передавать данные в фикстуру через параметризацию теста.
```suggestion
@pytest.fixture
def customer_data(request):
    params = getattr(request, "param")
    post_code = param.get("post_code")
    return {
        "first_name": name_generator(post_code),
        "last_name": param.get("last_name"),
        "post_code": post_code,
    }
```